README.rst

```diff
@@ -1,7 +1,7 @@
 pyvideo/data
 ===========
 
-*Python related videos and metadata*
+*Python related videos and metadata* for https://pyvideo.org
```

Comment:
```suggestion
*Python-related videos and metadata powering [PyVideo](https://pyvideo.org)
```

I suggest a slightly different phrasing that matches the current snippet description for this repository.